docs/usage_guide.rst

```diff
@@ -887,6 +939,21 @@ You can force a snapshot taking by clicking in the floppy disk icon at the top b
 
 Snapshots won't be saved if there is any error querying information for external sources. If you want to force the snapshot to be saved when an external source is reporting an error you can select the option `Ignore Errors`.
 
+It is possible to remove snapshots from the history in case the information is not correct or needs to be modified. To do it click in their point at the dashboard's graph and this will open a menu
+
+.. image:: images/delete_snapshot_menu.jpg
+   :alt: Delete snapshot
+   :align: center
+
+Clicking on delete will remove the saved information for that snapshot. The same menu allows to export the information for one snapshot to your system. Four files are then generated, two meant to be used if you want to import information about snapshots later and the other 2 have information in a human redeable format to be used in accounting. ``balances_snapshot``  contains information about the balances at the snapshot time for the different assets rotki knew you had and ``location_data_snapshot`` has the value per location of the assets you had.
+
+Finally information about snapshots can be imported back into the app using the files you exported with the sufix ``_import```. To import them use the import functionality in clicking on the third icon at the top bar.
```

Comment:
```suggestion
Finally information about snapshots can be imported back into the app using the files you exported with the suffix ``_import```. To import them use the import functionality by clicking on the save disk icon at the top bar.
```